src/Features/CSharp/Portable/CodeFixes/ImplementAbstractClass/ImplementAbstractClassCodeFixProvider.cs

```diff
@@ -73,7 +73,7 @@ public sealed override FixAllProvider GetFixAllProvider()
         // internal for testing purposes.
         internal static string GetCodeActionId(string assemblyName, string abstractTypeFullyQualifiedName)
         {
-            return CSharpFeaturesResources.ImplementAbstractClass + ";" +
+            return CSharpFeaturesResources.Implement_Abstract_Class + ";" +
```

Comment:
>Implement_Abstract_Class [](start = 43, length = 24)

Is there a reason why all words start with Caps here? your general pattern seems to be Caps for the first word in the sentence?